Cluster aware routers for typed (#27315)
akka/akka: akka-docs/src/main/paradox/typed/actor-discovery.md

## Cluster Receptionist

The `Receptionist` also works in a cluster, an actor registered to the receptionist will appear in the receptionist 
of the other nodes of the cluster.

> The state for the receptionist is propagated via @ref:[distributed data](../distributed-data.md) which means that each node 

Reviewer: Distributed Data

will eventually reach the same set of actors per `ServiceKey`.

`Subscription`s and `Find` queries to a clustered receptionist will keep track of cluster reachability and only list 
registered actors that are reachable. The full set of actors, including unreachable ones, is available through 
@scala[`Listing.allServiceInstances`]@java[`Listing.getAllServiceInstances`].

> One important difference from a local only receptions is the serialisation concerns, all messages sent to and back from 

Reviewer 2: ```suggestion
One important difference from local only receptions are the serialization concerns, all messages sent to and back from 
```

or
```suggestion
One important difference from a local only reception are the serialization concerns, all messages sent to and back from 
```
